server/neural_network_interface.py

```diff
@@ -46,14 +47,18 @@ def generate_move(self, sid, gesture_data, message_data):
 
         (flip, color, letter) = self._run_models(message)
```

Comment:
I realize this is technically a "my bad," but can we rename these to `flip_tuple` or `flip_tup` or something like that to emphasize they hold both a flip/color/letter value _and_ a "probability"?

Reply:
ack